scootapi/testhelpers/generators.go

```diff
@@ -32,7 +32,7 @@ func GenJobDefinition(rng *rand.Rand) *scoot.JobDefinition {
 func GenTask(rng *rand.Rand) *scoot.TaskDefinition {
 
 	cmd := scoot.NewCommand()
-	cmd.Argv = []string{"sleep", "1"}
+	cmd.Argv = []string{"sleep 500", "complete 0"}
```

Comment:
I think we need to be more opinionated and/or more configurable with regard to testing. Do you want to make the default runner defer to simExec? Or complete the TODO of adding configuration for runner and worker, and specify those in our smoketests?